packages/eslint-config-react-native-community/package.json

```diff
@@ -12,12 +12,12 @@
   "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/eslint-config-react-native-community#readme",
   "dependencies": {
     "@react-native-community/eslint-plugin": "^1.1.0",
-    "@typescript-eslint/eslint-plugin": "^4.22.1",
+    "@typescript-eslint/eslint-plugin": "^5.15.0",
     "@typescript-eslint/parser": "^4.22.1",
```

Comment:
`@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` should be on the same version

Reply:
thks for catching this